README.spanish.md

```diff
@@ -787,21 +836,21 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **TL;DR:** Con el ecosistema npm es común tener muchas dependencias para un proyecto. Las dependencias siempre deben mantenerse bajo control a medida que se encuentran nuevas vulnerabilidades. Usa herramientas como [npm audit](https://docs.npmjs.com/cli/audit) o [snyk](https://snyk.io/) para rastrear, monitorear y parchear dependencias vulnerables. Integra estas herramientas con su configuración de CI para que pueda detectar una dependencia vulnerable antes de que llegue a producción.
 
-**De lo contrario:** Un atacante podría detectar su marco web y atacar todas sus vulnerabilidades conocidas.
+**De lo contrario:** Un atacante podría detectar su framework web y atacar todas sus vulnerabilidades conocidas.
```

Comment:
marco === frame, marco de trabajo would be more accurate, but most people use the term untranslated